src/AnchoredOverlay/AnchoredOverlay.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import React, {useCallback, useRef, useState} from 'react'
+import {Args, Meta} from '@storybook/react'
+
+import {BaseStyles, Box, ThemeProvider} from '..'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import {Button} from '../Button'
+
+export default {
+  title: 'Components/AnchoredOverlay',
+  component: AnchoredOverlay,
+  args: {
+    'With Anchor?': true,
+    width: 'small',
+    height: 'auto',
+    portalContainerName: 'Portal name',
+    side: 'outside-bottom',
+  },
+  argTypes: {
+    'With Anchor?': {
+      type: 'boolean',
+    },
+    width: {
+      type: {
+        name: 'enum',
+        value: ['small', 'medium', 'large', 'xlarge', 'xxlarge', 'auto'],
+      },
+      portalContainerName: {type: 'string'},
+    },
+    height: {
+      type: {
+        name: 'enum',
+        value: ['xsmall', 'small', 'medium', 'large', 'xlarge', 'auto', 'initial'],
+      },
+      portalContainerName: {type: 'string'},
+    },
+    side: {
+      type: {
+        name: 'enum',
+        value: [
+          'inside-top',
+          'inside-bottom',
+          'inside-left',
+          'inside-right',
+          'inside-center',
+          'outside-top',
+          'outside-bottom',
+          'outside-left',
+          'outside-right',
+        ],
+      },
+    },
+  },
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
```

Comment:
```suggestion
```

We should be good to remove these decorators as they're loaded in by default. Let me know if that causes any problems so!